meson.build

```diff
@@ -154,18 +154,6 @@ endif
 
 enable_installed_tests = get_option('installed-tests')
 
-summary({
-    'Enable docbook documentation': build_docbook,
-    'Enable pipewire support': have_pipewire,
-    'Enable libsystemd support': have_libsystemd,
-    'Enable geoclue support': have_geoclue,
-    'Enable libportal support': have_libportal,
-    'Enable installed tests:': enable_installed_tests,
-  },
-  section: 'Optional builds',
-  bool_yn: true,
-)
```

Comment:
You do not strictly speaking need to move this around -- summary() can be called multiple times, and will merge identically named sections. Since the addition is defined in a subdir anyway, that might be fine to append there too.

Reply:
Thanks! I did have it that way at first but for some reason it didn't look/feel right so I moved it around instead. Happy to change it either way though.

Reply:
I don't feel strongly about this in either direction, so let's go with what @whot preferred.